app/views/events/index.html.erb

```diff
@@ -12,7 +12,7 @@
 
   function listenForPusherEvents() {
     var pusher = new Pusher('app_key', {
-      wsHost: '192.168.99.100',
+      wsHost: 'localhost',
```

Comment:
if this worked for you, we prob need to make it configurable. docker doesn't run on "localhost" for me